### DISCUSSION TITLE
Configure Home-Idp-Discovery Config with Realms Export

### DISCUSSION THREAD

**Discussion by HanifAbRazak** [sventorben/keycloak-home-idp-discovery#317] [Q&A]
I need help to preconfigure the config within realms export json file. Please advice.

Thank you. 

![image](https://github.com/sventorben/keycloak-home-idp-discovery/assets/142989669/bf6f9d6b-7736-4369-9ad4-1e8cbde7abfe)
![image](https://github.com/sventorben/keycloak-home-idp-discovery/assets/142989669/6b7e4be9-46cc-456b-a2e1-83895dcb029c)

**Comment by sventorben** [comment]
Hey @HanifAbRazak,

you will need to add an `authenticatorConfig` like this:

```json
{
      "id": "78baff39-dc8f-4c91-98e2-915102fa4529",
      "alias": "config",
      "config": {
        "forwardToFirstMatch": true,
        "bypassLoginPage": false,
        "forwardToLinkedIdp": false,
        "forwardUnverifiedEmail": false,
        "userAttribute": "email"
      }
    }
```

The authenticatorConfig is referenced via its `alias` by the `authenticatorConfig` property in the `authenticatorExecution`s

Please see the docs for supported options: https://sventorben.github.io/keycloak-home-idp-discovery/configuration.html#general-configuration-options
I have added the json property names in the table for reference.
